README.md

```diff
@@ -0,0 +1,1206 @@
+
+``` python
+us_pr_holidays = holidays.country_holidays('US', subdiv='PR')
+'2018-01-06' in us_holidays     # False
+'2018-01-06' in us_pr_holidays  # True
+```
+
+Please see the [holidays
+documentation](https://holidays.readthedocs.io/) for additional examples
+and detailed information.
+
+## Available Countries
+
+We currently support 158 country codes. The standard way to refer to a
+country is by using its [ISO 3166-1 alpha-2
+code](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes), the
+same used for domain names, and for a subdivision its [ISO 3166-2
+code](https://en.wikipedia.org/wiki/ISO_3166-2). Some countries have
+common or foreign names or abbreviations as aliases for their
+subdivisions. These are defined in the (optional) `subdivisions_aliases`
+attribute. Some of the countries support more than one language for
+holiday names output. A default language is defined by
+`default_language` (optional) attribute for each entity and is used as a
+fallback when neither user specified language nor user locale language
+available. The default language code is a [ISO 639-1
+code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). A list of
+all languages supported by country is defined by `supported_languages`
+(optional) attribute. If there is no designated [ISO 639-1
+code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) then [ISO
+639-2 code](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) can
+be used.
+
+Many countries have other categories of holidays in addition to common
+(national-wide) holidays: bank holidays, school holidays, additional
+(paid or non-paid) holidays, holidays of state or public employees,
+religious holidays (valid only for these religions followers). A list of
+all categories supported by country is defined by `supported_categories`
+(optional) attribute.
+
+The following is a list of supported countries, their subdivisions
+followed by their aliases (if any) in brackets, available languages and
+additional holiday categories. All countries support **PUBLIC** holidays
+category by default. All other default values are highlighted with bold:
+
+<table style="width: 100%">
+<colgroup>
+<col style="width: 20.0%" />
+<col style="width: 4.0%" />
+<col style="width: 46.0%" />
+<col style="width: 20.0%" />
+<col style="width: 10.0%" />
+</colgroup>
+<thead>
+<tr>
+<th><p>Country</p></th>
+<th><p>Code</p></th>
+<th><p>Subdivisions</p></th>
+<th><p>Supported Languages</p></th>
+<th><p>Supported Categories</p></th>
+</tr>
+</thead>
+<tbody>
+<tr class="row-even"><td style="white-space: normal; word-break: break-word;" ><p>Afghanistan</p></td>
+<td style="white-space: normal; word-break: break-word;" ><p>AF</p></td>
+<td style="white-space: normal; word-break: break-word;" ></td>
+<td style="white-space: normal; word-break: break-word;" ><p>en_US, <strong>fa_AF</strong>, ps_AF</p></td>
+<td style="white-space: normal; word-break: break-word;" ></td>
+</tr>
```

Comment:
It looks too cluttered. Try to use CSS-styles for td, th. And <p></p> tags are unnecessary in table .